### PR TITLE
add solution of 25 october issue #47

### PR DESCRIPTION
**Pull request by Nikhilsl12**
I have added the solution for 25th October and removed my previous pull request for 2 solutions. 

Screenshot (115)
Screenshot (116)

### CODE REVIEW

File: October 2024/Alternate_Sorting.cpp
Comment (Ayu-hack) [inline October 2024/Alternate_Sorting.cpp:1]: Great Work!
 Merged.
Thank you for contributing .
Feel free to tag me in your Hacktoberfest certificate or badges LinkedIn post! 😄 
👉 Follow my GitHub and connect with me on [LinkedIn](https://www.linkedin.com/in/ayush-tiwari02)!

**Comment by Ayu-hack** [comment]
Great Work!
 Merged.
Thank you for contributing .
Feel free to tag me in your Hacktoberfest certificate or badges LinkedIn post! 😄 
👉 Follow my GitHub and connect with me on [LinkedIn](https://www.linkedin.com/in/ayush-tiwari02)!

**Comment by Ayu-hack** [comment]
if possible do follow on [GitHub](https://github.com/Ayu-hack) and connect on [LinkedIn](https://www.linkedin.com/in/ayush-tiwari02) as a token of appreciation for the efforts! 🙌

**Comment by Nikhilsl12** [comment]
Yeah sure , can i ask you a question that at present my all 4 pr are done are all are on 7 days review period so does it gets confirmed till 31st.

**Comment by Ayu-hack** [comment]
Your PR should ACCEPTED before 31 oct.

**Pull request merged**